alerts/osd.libsonnet

```diff
@@ -68,6 +68,18 @@
               severity_level: 'warning',
             },
           },
+          {
+            alert: 'CephOSDLowSpace',
+            expr: |||
+              (ceph_osd_stat_bytes_used{%(cephExporterSelector)s} / ceph_osd_stat_bytes{%(cephExporterSelector)s}) * 100 > 85
```

Comment:
How its different from rule in https://github.com/ceph/ceph-mixins/blob/master/alerts/utilization.libsonnet?